hwy/ops/x86_128-inl.h

```diff
@@ -5592,6 +5592,24 @@ HWY_API Vec128<float, N> ConvertTo(Simd<float, N, 0> /* tag */,
   return Vec128<float, N>{_mm_cvtepi32_ps(v.raw)};
 }
 
+template <size_t N>
+HWY_API Vec128<float, N> ConvertTo(HWY_MAYBE_UNUSED Simd<float, N, 0> dd,
+                                   const Vec128<uint32_t, N> v) {
+#if HWY_TARGET <= HWY_AVX3
+  return Vec128<float, N>{_mm_cvtepu32_ps(v.raw)};
+#else
+  // Based on wim's approach (https://stackoverflow.com/questions/34066228/)
+  const Repartition<uint32_t, decltype(dd)> du32;
```

Comment:
This will work but `RebindToUnsigned<decltype(dd)>` would be a bit more clear and emphasizes we're not changing the number of lanes.